.travis.yml

```diff
@@ -47,6 +47,29 @@ matrix:
            PROJECT_URL=https://github.com/joblib/joblib.git
       python: 3.7
       if: commit_message =~ /(\[ci downstream\]|\[ci joblib\])/
+    - os: linux
+      env: PROJECT=ray TEST_REQUIREMENTS="setproctitle psutil"
+      python: 3.7
+      if: commit_message =~ /(\[ci downstream\]|\[ci ray\])/
+      install:
+        - pip install .
+        - pip install --upgrade -r dev-requirements.txt
+        - pip install $TEST_REQUIREMENTS;
+        - pip install ray==0.6.4;
+        - PROJECT_DIR="$(python -c "import os,ray,sys;print(os.path.dirname(ray.__file__));sys.stdout.flush()")";
+        - rm $PROJECT_DIR/cloudpickle/cloudpickle.py;
+        - cp cloudpickle/cloudpickle.py $PROJECT_DIR/cloudpickle/cloudpickle.py;
+        - PROJECT_DIR="$PROJECT_DIR/tests";
+        - pip list
+      script:
+        - COVERAGE_PROCESS_START="$TRAVIS_BUILD_DIR/.coveragerc" PYTHONPATH='.:tests' pytest -r s
+        - pushd $PROJECT_DIR
+        - pytest -vl --duration=10 $PROJECT_DIR/test_mini.py
+        - TEST_RETURN_CODE=$?
```

Comment:
The stuff with the `TEST_RETURN_CODE` is unnecessary. I think it's used in the other part because there's a big multi-line bash command.

Also, it's not working because the `$?` doesn't seem to work when it's part of a different line in the Travis file.